resources/server/sv_main.ts

```diff
@@ -42,19 +42,13 @@ function getRandomPhoneNumber() {
 }
 
 async function generatePhoneNumber(identifier: string) {
-  const _identifier = identifier;
-  let phoneNumber = await usePhoneNumber(_identifier);
-  let id = await getIdentifierByPhoneNumber(phoneNumber);
+  let phoneNumber = await usePhoneNumber(identifier);
 
-  do {
-    if (!phoneNumber) {
-      phoneNumber = await getRandomPhoneNumber();
-      const query = 'UPDATE users SET phone_number = ? WHERE identifier = ?';
-      await pool.query(query, [phoneNumber, _identifier]);
-    } else {
-      break;
-    }
-  } while (id);
+  if (!phoneNumber) {
+    phoneNumber = getRandomPhoneNumber();
```

Comment:
We should check that this number doesn't already exist

Reply:
Yea he did :p We all had discussed this in discord on the best way to implement it as the way it was before, for whatever reason, didnt work for everyone. I don't remember what the conclusion was though and i havent had time to implement it.

Reply:
Fixed

Reply:
@RockySouthpaw Did you add this back again? Or is this something we'll fix later? As this could be fixed in the PR, no?